pkg/config/integrations.go

```diff
@@ -16,30 +16,30 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-type integrationsVersion int
+type IntegrationsVersion int
```

Comment:
A number of things are now exposed outside this package for use in the converter code. Let's discuss if this goes too far here.